dvc/stage/serialize.py

```diff
@@ -70,16 +82,15 @@ def _serialize_params(params: List[ParamsDependency]):
         dump = param_dep.dumpd()
         path, params = dump[PARAM_PATH], dump[PARAM_PARAMS]
         if isinstance(params, dict):
-            k = list(params.keys())
+            k = sorted(params.keys())
```

Comment:
We were not sorting params keys for pipeline file before, only for the lock. Now, the same behavior is for lockfile.